modules/base/rendering/pointcloud/renderablepointcloud.cpp

```diff
@@ -435,6 +474,20 @@ RenderablePointCloud::ColorSettings::ColorSettings(const ghoul::Dictionary& dict
     }
     pointColor.setViewOption(properties::Property::ViewOptions::Color);
     addProperty(pointColor);
+
+    if (hasColoring) {
+        const Parameters::ColorSettings settings = *p.coloring;
+        enableOutline = settings.enableOutline.value_or(enableOutline);
+        addProperty(enableOutline);
+
+        outlineColor.setViewOption(properties::Property::ViewOptions::Color);
+        outlineColor = settings.outlineColor.value_or(outlineColor);
+        addProperty(outlineColor);
+
+        outlineWeight = settings.outlineWeight.value_or(outlineWeight);
+        addProperty(outlineWeight);
```

Comment:
Agreed!  Move the reading of the values to the if statement above, and always add the properties. 

Otherwise, the user won't be able to experiment with borders if no coloring is specified in the asset, which is very confusing and faulty behavior.